[2022-12-16 13:03:31,447] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T11:55:00+00:00 [queued]>
[2022-12-16 13:03:31,558] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T11:55:00+00:00 [queued]>
[2022-12-16 13:03:31,559] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 13:03:31,559] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-16 13:03:31,559] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 13:03:31,692] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-12-16 11:55:00+00:00
[2022-12-16 13:03:31,707] {standard_task_runner.py:52} INFO - Started process 3895 to run task
[2022-12-16 13:03:31,731] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-12-16T11:55:00+00:00', '--job-id', '1930', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpoj05uo77', '--error-file', '/tmp/tmpd7byy4pl']
[2022-12-16 13:03:31,762] {standard_task_runner.py:80} INFO - Job 1930: Subtask get_last_id
[2022-12-16 13:03:32,316] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T11:55:00+00:00 [running]> on host 8e2f65ec4c74
[2022-12-16 13:03:32,991] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T11:55:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-16T11:55:00+00:00
[2022-12-16 13:03:33,107] {base.py:68} INFO - Using connection ID 'gp_conn' for task execution.
[2022-12-16 13:03:33,334] {python.py:173} INFO - Done. Returned value was: None
[2022-12-16 13:03:33,936] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221216T115500, start_date=20221216T130331, end_date=20221216T130333
[2022-12-16 13:03:34,065] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-16 13:03:34,342] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
